__tests__/cron/cleanZombieUserCompany.ts

```diff
@@ -0,0 +1,60 @@
+import { cleanZombieUserCompany as cron } from '../../src/cron/cleanZombieUserCompany';
+import { saveFixtures } from '../helpers';
+import { expectSuccessfulCron } from '../helpers';
+import { User } from '../../src/entity/user/User';
+import { UserCompany } from '../../src/entity/UserCompany';
+import { usersFixture } from '../fixture/user';
+import { DataSource } from 'typeorm';
+import createOrGetConnection from '../../src/db';
+import { subHours } from 'date-fns';
+
+let con: DataSource;
+
+beforeAll(async () => {
+  con = await createOrGetConnection();
+});
+
+beforeEach(async () => {
+  await saveFixtures(con, User, usersFixture);
+  await saveFixtures(con, UserCompany, [
+    {
+      userId: '1',
+      verified: true,
+      email: 'u1@com1.com',
+      code: '123',
+    },
+  ]);
+});
+
+describe('cleanZombieUserCompany', () => {
```

Comment:
I see you registered it but add `should be registered` test as well (example in daily digest cron)

Reply:
Added registered test too